src/ucp/wireup/select.c

```diff
@@ -1946,16 +1947,12 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
         UCS_STATIC_BITMAP_RESET_ALL(&tl_bitmap);
 
         ucs_memory_type_for_each(mem_type) {
-            UCP_CONTEXT_MEM_CAP_TLS(context, mem_type, reg_mem_types,
-                                    mem_type_tl_bitmap);
+            ucp_wireup_memaccess_bitmap(context, mem_type, &mem_type_tl_bitmap);
```

Comment:
don't we want to select only those tls which support registration ?

Reply:
if the md does not rquire memh/rkey (like self/tcp/cma) we don't need to require registration